renderer/nyaovim-app.ts

```diff
@@ -164,6 +164,117 @@ Polymer({
                     client.command(c);
                 }
             });
+
+            ipc.on('nyaovim:copy', (_: Electron.IpcRendererEvent) => {
+                // get current vim mode
+                var m = client.commandOutput('echo mode()');
```

Comment:
I think `client.eval('mode()')` should also work and prefers to `commandOutput()` for this purpose.